src/utils.js

```diff
@@ -72,71 +73,61 @@ const _runJavascript = (jsString, window, filename = 'script', lineOffset = 0, c
 module.exports._runJavascript = _runJavascript;
 
 /**
- * Convert (normalize prototype) native object to pass to JS.
- * Or convert JS object when passing to native.
- * Node's vm module was slow so there's vm-one for one-way bindings.
- * github.com/modulesio/vm-one
+ * Normalize prototype between native and V8.
+ * Node's vm module was slow so there's vm-one for one-way bindings:
+ *   github.com/modulesio/vm-one
  * nativeVm.setPrototype will do the nan casting setPrototype between types.
  * Required for instanceof to work.
  *
  * @param obj - Object to convert.
  * @param targetContext - window or vm context that prototype will get retrieved from.
  */
-const _normalizeBuffer = (function () {
-  let dummyWindow;
+const _normalizePrototype = (obj, targetContext) => {
+  let name = obj && obj.constructor && obj.constructor.name;
+  if (obj instanceof targetContext[name]) { return obj; }
 
-  return (obj, targetContext) => {
-    // Dummy window to reference original prototypes
-    // (e.g., in case Promise gets overwritten by polyfill).
-    if (!dummyWindow) {
-      let vmo = GlobalContext.nativeVm.make();
-      dummyWindow = vmo.getGlobal();
-      dummyWindow.vm = vmo;
-    }
+  const isToWindow = !!targetContext[symbols.prototypesSymbol];
 
-    const name = obj && obj.constructor && obj.constructor.name;
+  // Use Buffer for Blob.
+  if (name === 'Blob') { name = 'Buffer'; }
```

Comment:
Is this the same thing the previous code was doing?

`Blob` is not itself a `Buffer`, but it does have a `Buffer` under `Blob.buffer` -- _and_ that is what needs a prototype conversion, not the `Blob`.

Reply:
Previous code: https://github.com/webmixedreality/exokit/pull/441/files#diff-2b4ca49d4bb0a774c4d4c1672d7aa781L96